src/track/trackmetadatataglib.cpp

```diff
@@ -838,6 +838,29 @@ void readTrackMetadataFromID3v2Tag(TrackMetadata* pTrackMetadata,
         parseTrackPeak(pTrackMetadata,
                 toQString(pTrackPeakFrame->fieldList()[1]));
     }
+    TagLib::ID3v2::FrameList ratingFrame = tag.frameListMap()["POPM"];
+    int rating = 0;
+    if(!ratingFrame.isEmpty()) {
+        // RatingString "traktor@native-instruments.de rating=255 counter=2"
+
+        QString sRating = TStringToQString(ratingFrame.front()->toString());
+        sRating = sRating.section("=",1,2).left(3);
+        float fRating = sRating.toInt();
+        rating = ceil(fRating/51);
```

Comment:
well, this is a problem then. How does the code decide which rating to use for its star rating display? and as far as i know, metatags are only read in on a forced reload or new file.. a library scanner didn't even refresh the rating.  How do you store a Qmap/hash in the database? how would you use the library search to search for rated files in stars, when its not stored in stars?